app/services/food_service.rb

```diff
@@ -0,0 +1,14 @@
+class FoodService
+  def self.conn
+    Faraday.new(url: "https://api.nal.usda.gov")
+    binding.pry
+  end
+
+  def self.get_all_food_data(search_params)
+    response = conn.get("/fdc/v1/foods/search?api_key=#{ENV['food_api_key']}&query=#{search_params}")
+    # response =  Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/foods/search?api_key=R0usgIZUXL2umn3LYnclsi1juWdnKOTvZuAGKzPN&query=#{search_params}")
```

Comment:
This looks like a great start. I love seeing that you have some `prys` going on. What did you find you have access to here?